modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -858,7 +1004,7 @@ FeatureFloodCount::appendPeriodicNeighborNodes(FeatureData & data) const
     {
       Elem * elem = _mesh.elemPtr(entity);
 
-      for (unsigned int node_n = 0; node_n < elem->n_nodes(); node_n++)
+      for (auto end = elem->n_nodes(), node_n = 0u; node_n < end; ++node_n)
```

Comment:
Just experimenting. When using auto to declare multiple variables, they must match types. I'll change this one since I don't like the way it looks either.